ampersand-search-select.js

```diff
@@ -0,0 +1,241 @@
+;(function() {
+  var d3 = require('d3');
+  var _ = require('lodash');
+
+  var AmpersandState = require('ampersand-state');
+  var AmpersandView = require('ampersand-view');
+  var AmpersandSubCollection = require('ampersand-subcollection');
+
+  var SearchSelectState = AmpersandState.extend({
+    props: {
+      selected: 'object'
+    },
+    session: {
+      query: [ 'string', false, ''],
+      data: 'object',
+      idAttribute: 'string',
+      imageAttribute: 'string',
+      queryAttribute: 'string',
+
+      // Private Variables
+      _data: 'object',
```

Comment:
Sorry, I didn't mean sticking it in `props`, but sticking it directly on the object like:
```js
var SearchSelectState = AmpersandState.extend({
  _data: {}
});